docker-compose.yml

```diff
@@ -60,6 +60,13 @@ services:
       << : *default-healthcheck
       test: "echo 'db.stats().ok' | mongo localhost:27017/test --quiet"
 
+  mongo-2.6:
+    image: mongo:2.6.12
```

Comment:
Should this be just 2.6 and use the latest patch version (as per Mongo 3.4)

Reply:
Don't see a problem with this 👍